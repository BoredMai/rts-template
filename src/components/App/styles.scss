:local(.app-div) {
  text-align: center;
  // add your styles here!
}
