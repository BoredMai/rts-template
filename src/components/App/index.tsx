import React, { Component } from 'react';

import { appDiv } from './styles.scss';

class App extends Component<{}, {}> {
  render(): JSX.Element {
    return (
      <h1 className={appDiv}>
        This is the main App component. Start editing from here!
      </h1>
    );
  }
}

export default App;
