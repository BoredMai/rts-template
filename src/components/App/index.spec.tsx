import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import App from '.';

let wrapper: ShallowWrapper<{}, {}, App>;

describe('<App>', () => {
  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it('renders properly', () => {
    expect(wrapper.getElement()).toMatchSnapshot();
  });
});
