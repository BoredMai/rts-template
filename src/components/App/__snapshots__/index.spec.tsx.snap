// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> renders properly 1`] = `
<h1
  className="appDiv"
>
  This is the main App component. Start editing from here!
</h1>
`;
